packages/php-wasm/logger/src/logger.ts

```diff
@@ -1,14 +1,27 @@
-import { UniversalPHP } from '@php-wasm/universal/src/lib/universal-php';
+import {
+	PHPRequestErrorEvent,
+	UniversalPHP,
+} from '@php-wasm/universal/src/lib/universal-php';
 /**
  * Log severity levels.
  */
-export type LogSeverity = 'debug' | 'info' | 'warn' | 'error' | 'fatal';
+export type LogSeverity = 'Debug' | 'Info' | 'Warn' | 'Error' | 'Fatal';
+
+/**
+ * Log prefix.
+ */
+export type LogPrefix = 'Playground' | 'PHP-WASM';
 
 /**
  * A logger for Playground.
  */
-export class Logger {
-	private readonly LOG_PREFIX = 'Playground';
+export class Logger extends EventTarget {
```

Comment:
Note this will require the following addition to the main file in this package (index.ts?)

```ts
// PHP.wasm requires WordPress Playground's Node polyfills.
import '@php-wasm/node-polyfills';
```

Reply:
[8f5dab5](https://github.com/WordPress/wordpress-playground/pull/1095/commits/8f5dab5518fb8a6c44e85db093dd03f51acec0a5)